collect-settings.rst

```diff
@@ -93,6 +101,16 @@ Form Management Settings
 - :guilabel:`Default to finalized` When enabled, forms are automatically finalized upon reaching the end of the form. You can opt out of finalizing any form when completing it.
 - :guilabel:`Constraint processing` used to select when response constraints are validated: After each swipe, or at finalization.
 - :guilabel:`High res video` enables high-resolution recordings.
+- :guilabel:`Image size` is used to set an image size for all the widgets.There are five size options available as shown below:
```

Comment:
Instead of a note, use the version added directive.

http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded